webknossos/tests/test_generated_client.py

```diff
@@ -98,7 +98,9 @@ def test_user_list(auth_client: Client) -> None:
 
 
 @pytest.mark.vcr()
-def test_dataset_info(client: Client) -> None:
+def test_dataset_info() -> None:
+    with webknossos_context():
```

Comment:
naming is ambiguous, change defaults to fallbacks

Reply:
I'll adapt this in #577